setup.cfg

```
@@ -62,3 +62,6 @@ test =
     matplotlib>=2.2.4
     numpy
     pytest>=6.0.0
+    pytest-asyncio
+visual_tests =
```

```suggestion
visual_test =
```

To be consistent with the tests `test` extra